server/settings.py

```diff
@@ -185,6 +185,9 @@ def env(variable, fallback_value=None):
 S3_USE_HTTPS = False
 FLASK_ASSETS_USE_S3 = False
 USE_S3 = FLASK_ASSETS_USE_S3
+RECAPTCHA_PUBLIC_KEY = ''
+RECAPTCHA_PRIVATE_KEY = ''
```

Comment:
it's easier later to assign value if you use the `env()` function. No need to declare empty string as default